examples/standalone/workloads.tf

```
@@ -515,7 +494,7 @@ module "dlp_scanner" {
 */
 module "pubsub_to_bigquery" {
   source  = "terraform-google-modules/pubsub/google"
-  version = "~> 5.0"
+  version = "8.0"
```

```suggestion
  version = "~> 8.0"
```

please keep the `~>` where it is been used.
It means that it will be able to get minor updates and fixes but not major ones